awx/settings/defaults.py

```diff
@@ -869,6 +869,7 @@
         'awx.main.tasks': {'handlers': ['task_system', 'external_logger', 'console'], 'propagate': False},
         'awx.main.analytics': {'handlers': ['task_system', 'external_logger', 'console'], 'level': 'INFO', 'propagate': False},
         'awx.main.scheduler': {'handlers': ['task_system', 'external_logger', 'console'], 'propagate': False},
+        'awx.main.utils.external_logging': {'handlers': ['console']},  # force logging to console, if we log while trying to configure logging
```

Comment:
We have an existing method of handling these, by filtering them out (in the filter class).

https://github.com/ansible/awx/blob/284bd8377a2be85ab0ab163e9bd4d670be5085d0/awx/main/constants.py#L73-L81

If you want to _instead of this_ do it by the logging configuration, then go ahead. But I don't think this would work as-is. It would need `'propagate': False` if the intent was to avoid infinite logging loops. Also, in many cases we still want these in the file-based logs (although sometimes we don't), and that can get messier than what you're looking for.

Reply:
Good call out, I didn't realize we already had infrastructure in place to exclude things like this. I'll just add the utils module to that list instead.